azkaban-common/src/main/java/azkaban/AzkabanCommonModule.java

```diff
@@ -83,6 +86,9 @@ protected void configure() {
     bind(DataSource.class).to(AzkabanDataSource.class);
     bind(ExecutorManager.class).in(Scopes.SINGLETON);
     bind(AlerterHolder.class).in(Scopes.SINGLETON);
+    bind(CommonMetrics.class).in(Scopes.SINGLETON);
```

Comment:
Would it be a good idea to standardize on the @Singleton annotation? Not in this PR, but in the future.